tests/test_browsing_context.py

```diff
@@ -254,12 +253,11 @@ async def test_browsingContext_getTreeWithNestedCrossOriginContexts_contextsRetu
     } == result
 
 
-# TODO(sadym): make offline.
 @pytest.mark.asyncio
 async def test_browsingContext_afterNavigation_getTreeWithNestedCrossOriginContexts_contextsReturned(
         websocket, context_id):
-    nested_iframe = 'https://example.com/'
-    another_nested_iframe = 'https://example.org/'
+    nested_iframe = 'about:blank'
+    another_nested_iframe = 'about:about'
```

Comment:
Are they considered to be cross-origin?

Reply:
It it's a requirement, I'll revert it just to be safe.